src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -762,17 +762,28 @@ private static ImmutableArray<string> GetAndTestFixableDiagnosticIds(CodeFixProv
                 {
                     var mutableMap = new Dictionary<DiagnosticId, List<CodeFixProvider>>();
 
-                    foreach (var fixer in languageKindAndFixers.Value)
+                    foreach (var lazyFixer in languageKindAndFixers.Value)
                     {
-                        foreach (var id in this.GetFixableDiagnosticIds(fixer.Value, extensionManager))
+                        CodeFixProvider fixer;
+                        try
+                        {
+                            fixer = lazyFixer.Value;
+                        }
+                        catch
+                        {
+                            // Gracefully handle exceptions in creating fixer instance.
+                            continue;
```

Comment:
can we log/diagnostic (i.e. an AD diagnostic) these or yellow-bar somehow?  Having stuff fail to create and trying to track it down is a nightmare at times.

Reply:
Sure, I think we can show an infobar with type name and exception stack trace. Let me add that part.

Reply:
Done. Now we show the below:

![image](https://user-images.githubusercontent.com/10605811/87610554-8a04ab80-c6ba-11ea-8caa-a2fb6e378df2.png)

![image](https://user-images.githubusercontent.com/10605811/87610561-8ec95f80-c6ba-11ea-8588-fe4e08d3c602.png)